tests/acceptance/features/steps/nid.py

```diff
@@ -0,0 +1,195 @@
+
+        if len(subexpected) == len(subresult):
+            for i in range(len(subresult)):
+                try:
+                    int(subresult[i])
+                except ValueError:
+                    result = False
+                    print("The data {} is not a digit".format(subresult[i]))
+        else:
+            result = False
+    else:
+        result = False
+
+    return result
+
+
+@given(u'a connectivity to the FIWARE Catalogue')
+def step_check_availability_catalogue(context):
+    code = urllib.urlopen("http://catalogue.fiware.org/").getcode()
+
+    assert_that(code, equal_to(200), "FIWARE Catalogue is not responding.")
+
+
+@when(u'I execute the getnid with the following values')
+def step_obtain_list_ge_nids(context):
+    context.value = dict()
+
+    if context.table is not None:
+        for row in __dataset_utils__.prepare_data(context.table):
+            if 'type_value' in row.headings and row['type_value'] != 'False':
+                type = row['type_value']
+
+                command = '../../glancesync/getnid.py --type {}'.format(type)
+
+                context.value[type] = cmd(command).replace('\n', '').replace('\r', '').replace(" ", "")
+                context.type = type
+            elif 'wikitext' in row.headings:
+                command = '../../glancesync/getnid.py --wikitext'
+
+                context.value['wikitext'] = cmd(command).replace('\n', '').replace('\r', '').replace(" ", "")
+
+
+@then(u'I obtain the following list of nid corresponding to that chapter')
+def step_check_data_recover(context):
+    if context.table is not None:
+        for row in __dataset_utils__.prepare_data(context.table):
+            if 'chapter_value' in row.headings:
+                resource = row['resources_value']
+
+        # Load the content expected from file
+        expected_value = __load_response__(resource)
+
+    assert_that(context.value[context.type], equal_to(expected_value), "Response obtained from FIWARE Catalogue is not "
+                                                                       "the expected one.")
+
+
+@given(u'that I have the gitnid application installed')
+def step_impl_check_application_installed(context):
+    # Check the availability of the gitnid.py code
+    result = os.path.isfile("../../glancesync/getnid.py")
+
+    assert_that(result, equal_to(True), "The python scripts does not exist in the folder ../../glancesync/getnid.py")
+
+
+@when(u'I execute the gitnid application with the option')
+def step_impl_execute_with_options(context):
+    if context.table is not None:
+        for row in __dataset_utils__.prepare_data(context.table):
+            if 'option_value' in row.headings:
+                option = row['option_value']
+
+                command = '../../glancesync/getnid.py {}'.format(option)
+
+                context.result = cmd(command)
+
+                assert_that(command, is_not(''))
```

Comment:
I think you mean **context.result**.
Add a custom message to the _assert_ to be used when it fails.